### ISSUE TITLE
twisted/conch/avatar.py doesn't match twisted coding standards

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7526]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Alex reported|
|-|-|
|Trac ID|trac#7526|
|Type|enhancement|
|Created|2014-06-18 16:55:27Z|
|Branch|https://github.com/twisted/twisted/tree/small-conch-cleanup-7526|

Notably:

* It uses implicit relative imports
* It uses `zope.interface.implements` instead of `implementer`
* It fails to place a ` ` between a `,` and an argument in a function call.

(Marking as a Python3 ticket due to the first two points)

<details><summary>Searchable metadata</summary>

```
trac-id__7526 7526
type__enhancement enhancement
reporter__Alex Alex
priority__normal normal
milestone__Python_3_x Python-3.x
branch__branches_small_conch_cleanup_7526 branches/small-conch-cleanup-7526
branch_author__cyli cyli
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1403110527000000 1403110527000000
changetime__1403151260000000 1403151260000000
version__None None
owner__cyli cyli

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|Alex commented|
|-|-|

This is resolved by: https://github.com/twisted/twisted/pull/29

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)|@cyli set owner to @cyli|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_3"></a>|@cyli commented|
|-|-|

(In [42792]) Branching to small-conch-cleanup-7526.

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_4"></a>|@cyli commented|
|-|-|

Thanks for working on this Alex!  I added a misc topfile corresponding to this ticket number, other than that it looks great.

Here are the buildbot test results:  https://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/small-conch-cleanup-7426

(Ignore the bad ticket number on that link - I previously created a branch with the wrong ticket number, but it was the same cherry-picked commit).

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_5"></a>|@cyli set status to `closed`|
|-|-|

(In [42795]) Merge small-conch-cleanup-7526: Small twisted coding-standard changes in conch

Author: Alex
Reviewers: cyli
Fixes: [#7526](https://github.com/twisted/twisted/issues/7526)

A couple of small changes to make twisted.conch.avatar more compliant
with the twisted coding standard.  This also happens to make it more
python3-compatible.
